doc/release-notes/release-7.sgml.in

```diff
@@ -57,6 +58,23 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 
 </descrip>
 
+<sect1>Removed purge tool
+<p>The <em>purge</em> tool (also known as <em>squidpurge</em>, and <em>squid-purge</em>)
+   was limited to managing UFS/AUFS/DiskD caches.
+
+<p>The cache contents display and search it provided can be obtained with a script
+   searching the cache manager <em>objects</em> report.
+
+<p>This tool used the custom <em>PURGE</em> HTTP method to remove cache
+   objects. This can be performed directly on any Squid configured to allow
+   the method. Like so:
+<verb>
+    acl PURGE method PURGE
+    http_access allow manager PURGE
```

Comment:
The `manager` part is a bug. Moreover, the http_access part is not needed at all (it is the existence of the `acl` line that matters).

We do not have to put any other ACL in an obviously incomplete example:

```suggestion
    ... and http_access allow PURGE traffic as/if needed ...
```

but if you want to put something (for the false sense of "extra safety"), we can put `localhost`, I guess:

```suggestion
    http_access allow localhost PURGE
```